plugins/services/src/js/utils/__tests__/TaskTableUtil-test.js

```diff
@@ -52,7 +54,7 @@ describe('TaskTableUtil', function () {
       }
     });
 
-    this.sortFunction = TaskTableUtil.getSortFunction('name');
+    this.getComparator = TaskTableUtil.getSortFunction('name');
```

Comment:
I believe that changing incrementally as we go is a better option at the moment as developers work on files that needs to be refactor in a way to make more readable but I'm also fine removing from this PR and creating a Jira ticket to refactor the naming.